xen/qa7/syn/readme.md

```diff
@@ -0,0 +1,111 @@
+# QMTech Artix-7 DDR3 board 
+
+## Specification
+
+* On-Board FPGA: XC7A35T-1FTG256C;
+* On-Board FPGA external crystal frequency: 50MHz;
+* XC7A35T-1FTG256C has rich block RAM resource up to 1,800Kb;
+* XC7A35T-1FTG256C has 33,280 logic cells;
+* On-Board N25Q064 SPI Flash，8M bytes for user configuration code;
+* On-Board 256MB Micron DDR3，MT41J128M16-15E;
+
+## Site URL
+  https://www.aliexpress.com/item/32964497318.html
+
+## Documentation, code
+  https://github.com/ChinaQMTECH/QM_XC7A35T_DDR3
+
+# Pi 1000 digits test
+
+| CPU   | Ticks (50MHz) |
+|-------|---------------|
+| vm1   |  1173         |
+| vm2   |   224         |
+| lsi   |   568         |
+| am4   |   308         |
+
+# Report Instance Areas: 
+## VM1 
+
+	+------+----------------------+-------------+------+
+	|      |Instance              |Module       |Cells |
+	+------+----------------------+-------------+------+
+	|1     |top                   |             |  3182|
+	|2     |  cpu                 |wbc_vm1      |  3109|
+	|3     |    cpu               |vm1_wb       |  1760|
+	|4     |      timer           |vm1_timer    |   151|
+	|5     |      vreg_rm         |vm1_reg_ram  |   232|
+	|6     |        vm1_vcram_reg |vm1_vcram    |   226|
+	|7     |    mem               |wbc_mem      |   931|
+	|8     |      ram             |qa7_mem      |   928|
```

Comment:
It's a real mess - ~1K LUTS for memory block. Synthesis definitely goes wrong

Reply:
It was implemented as "distributed" ram, so seems OK.
I got BRAM based implementation of `wbc_mem`, however I'm not able to get it implemented for 90MHz target, only 85MHz